### ISSUE TITLE
Error in test_address_ru.rb

### ISSUE THREAD

**Issue by gustavor-souza** [ffaker/ffaker#229]
this occurs by chance when runs rake test,  seems "Орёл" does not match with the regex

```
Failure:
  </\A[a-zа-я\-\s]+\z/i> expected to be =~
  <"Орёл">.
test_city(TestAddressRU)
/home/gustavo/Documents/Projects/ffaker/test/test_address_ru.rb:12:in `test_city'
      9:   end
     10: 
     11:   def test_city
  => 12:     assert_match /\A[a-zа-я\-\s]+\z/i, @tester.city
     13:   end
     14: 
```

**Comment by marocchino** [comment]
Oops, Do you have valid regexp for russian words?
because, I am not native in Russian. >_<

**Comment by isolo** [comment]
fixed in #230

**Comment by marocchino** [comment]
Thanks!
